packages/ckeditor5-package-generator/tests/index.js

```diff
@@ -99,89 +115,124 @@ describe( 'lib/index', () => {
 		expect( index ).to.be.a( 'function' );
 	} );
 
-	it( 'passes correct arguments to the validatePackageName()', async () => {
+	it( 'creates logger with verbose option set to false', async () => {
 		await index( packageName, options );
 
 		expect( stubs.validatePackageName.callCount ).to.equal( 1 );
 		expect( stubs.validatePackageName.getCall( 0 ).args[ 0 ].constructor.name ).to.equal( 'Logger' );
-		expect( stubs.validatePackageName.getCall( 0 ).args[ 1 ] ).to.equal( '@scope/ckeditor5-feature' );
+		expect( stubs.validatePackageName.getCall( 0 ).args[ 0 ] ).to.deep.equal( { verbose: true, info: stubs.logger.info } );
 	} );
 
-	it( 'passes undefined to validatePackageName() when the package name is undefined', async () => {
-		await index( undefined, options );
+	it( 'creates logger with verbose option set to true', async () => {
```

Comment:
Same as above, when it uses `true` and why this test checks anything related to the `validatePackageName()` function?